.travis.yml

```diff
@@ -22,6 +27,7 @@ notifications:
 
 before_script:
   - "flake8 ."
+  - if [ $GUI ]; then export DISPLAY=:99.0 && sh -e /etc/init.d/xvfb start -screen 0 1920x1080x24 && sleep 3; fi
 
 script:
-  - "py.test --capture=sys --showlocals --tb=long foxpuppet/tests "
\ No newline at end of file
+  - "py.test --capture=sys --showlocals --tb=long --driver Firefox --firefox-path=firefox/firefox-bin foxpuppet/tests"
```

Comment:
We should be using Tox here 😆 so change this to simply `tox`. No need for the path to Firefox as we've made sure it's in the path.